apps/condo/domains/common/components/UseDeskWidget.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import getConfig from 'next/config'
+
+const UseDeskWidget: React.FC = () => {
+
+    const { publicRuntimeConfig:{ UseDeskWidgetId } } = getConfig()
```

Comment:
Why is the config access method located in render section? May be move this line out of component body?